src/Compiler/ReleaseNotesBuilder.cs

```diff
@@ -33,10 +32,10 @@ public static string LabelPrefix
 
         public async Task<string> BuildReleaseNotes()
         {
-            LoadMilestones();
+            milestones = await gitHubClient.GetMilestones();
 
             GetTargetMilestone();
-            var issues = await GetIssues(targetMilestone);
```

Comment:
Above method GetTargegMilestone sounds like a side effect free method but does assign magically a field. Like above I would return the milestone. Then this method used mostly locals and we can ditch the fields